### PR TITLE
Added NFD known issue

### PR DESCRIPTION
**Pull request by ahardin-rh**
@mffiedler @wabouhamad PTAL

@vikram-redhat FYI

Preview Build: http://file.rdu.redhat.com/~ahardin/01222020/nfd-known-issue/release_notes/ocp-4-3-release-notes.html#ocp-4-3-known-issues

### CODE REVIEW

File: release_notes/ocp-4-3-release-notes.adoc [hunk 1]
```diff
@@ -1439,3 +1439,7 @@ $ oc get pods -n openshift-cluster-node-tuning-operator
 +
 Verify that the Pods are now in a `Running` state.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1791916[*BZ#1791916*])
+
+* The Node Feature Discovery (NFD) Operator version 4.2 is failing to deploy in
```
Comment (wabouhamad) [inline release_notes/ocp-4-3-release-notes.adoc]: version 4.3

Comment (wabouhamad) [reply release_notes/ocp-4-3-release-notes.adoc]: fails to deploy from OpertorHub on OpenShift Console.

As a workaround, download the oc client for your operating system, and place the kubeconfig from the installer in ~/.kube/config.  Follow these steps to deploy NFD Operator from CLI and github:

cd $GOPATH/src/openshift
git clone https://github.com/openshift/cluster-nfd-operator.git
cd cluster-nfd-operator
git checkout namespace-scoped
PULLPOLICY=Always make deploy
oc get pods -n openshift-nfd

Example:

# oc get pods -n openshift-nfd
NAME                           READY   STATUS    RESTARTS   AGE
nfd-master-gj4bh               1/1     Running   0          9m46s
nfd-master-hngrm               1/1     Running   0          9m46s
nfd-master-shwg5               1/1     Running   0          9m46s
nfd-operator-b74cbdc66-jsgqq   1/1     Running   0          10m
nfd-worker-87wpn               1/1     Running   2          9m47s
nfd-worker-d7kj8               1/1     Running   1          9m47s
nfd-worker-n4g7g                1/1     Running   1          9m47s

**Review by wabouhamad — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-3-release-notes.adoc (hunk 1 above)
Comment (wabouhamad) [reply release_notes/ocp-4-3-release-notes.adoc]: @zvonkok @ahardin-rh PTAL

**Comment by ahardin-rh** [comment]
@wabouhamad @zvonkok Thanks! Updated. PTAL. Preview build:
http://file.rdu.redhat.com/~ahardin/01222020/nfd-known-issue/release_notes/ocp-4-3-release-notes.html#ocp-4-3-known-issues

**Comment by wabouhamad** [comment]
@ahardin-rh  LGTM.  But I prefer we mention repo branch 

`release-4.3` instead of `namespace-scoped` once @zvonkok ports the fix to that branch.

**Pull request merged**